client/src/components/ActiveChat/Messages.js

```diff
@@ -8,20 +8,24 @@ const Messages = (props) => {
 
   return (
     <Box>
-      {messages.map((message) => {
-        const time = moment(message.createdAt).format('h:mm');
+      {messages
+        .sort(function (a, b) {
+          return a.id - b.id;
+        })
```

Comment:
This seems to sort the messages without breaking the conversation order; however, it has two concerns:

1. We can not assume the ids will be chronological. Can you use another field from the table instead of ids?
2. The operation is inefficient as it runs for every component re-render. Ideally, we want the operation to be performed when the data is fetched from the API.

Can you apply the above hints and address the bug correctly?